packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -1357,31 +1177,19 @@ export namespace EmrCreateCluster {
      * The Amazon EC2 Availability Zone for the cluster. AvailabilityZone is used for uniform instance groups, while AvailabilityZones
      * (plural) is used for instance fleets.
      *
-     * @default EMR selected default
+     * @default - EMR selected default
      */
     readonly availabilityZone?: string;
 
     /**
      * When multiple Availability Zones are specified, Amazon EMR evaluates them and launches instances in the optimal Availability Zone.
      * AvailabilityZones is used for instance fleets, while AvailabilityZone (singular) is used for uniform instance groups.
      *
-     * @default EMR selected default
+     * @default - EMR selected default
      */
     readonly availabilityZones?: string[];
```

Comment:
Can these just be one property?

Reply:
I don't see why not. It was not the scope of this change, but I've made a note of it as well.